src/main/java/fr/openmc/core/features/contest/managers/ContestPlayerManager.java

```diff
@@ -31,23 +32,22 @@ public void setPointsPlayer(Player player, int points) {
     }
 
     public String getTitleWithPoints(int points) {
-        int[] pointsRank = {10000, 2500, 2000, 1500, 1000, 750, 500, 250, 100, 0};
-        String[] categories = {
-                "Dictateur en ",
-                "Colonel en ",
-                "Addict en ",
-                "Dieu en ",
-                "Légende en ",
-                "Sénior en ",
-                "Pro en ",
-                "Semi-Pro en ",
-                "Amateur en ",
-                "Noob en "
+        Object[][] ranks = {
```

Comment:
Utiliser une map au lieu de faire un tableau a 2 dimensions, cela sera plus facile a manipuler 
`Map.of()`